lib/mosaic-network/src/main/java/org/eclipse/mosaic/lib/coupling/AbstractNetworkAmbassador.java

```diff
@@ -438,7 +445,6 @@ private synchronized void process(VehicleUpdates interaction) throws InternalFed
             latestVehicleUpdates = interaction;
 
             if (!interaction.getAdded().isEmpty()) {
-                log.debug("Add Vehicles at first movement");
                 List<VehicleData> addedVehicles = interaction.getAdded();
                 Comparator<UnitData> comp = new UnitNameComparator();
                 addedVehicles.sort(comp);
```

Comment:
Line 456 not needed to be warn, just debug:
`log.warn("Vehicle with ID {} is not in the registered list (no config arrived yet), ignoring entry.", vi.getName());`
better:
`log.debug("Vehicle with ID {} is not in the registered list (no config arrived yet)", vi.getName());`

Reply:
done in https://github.com/eclipse/mosaic/pull/138/commits/13dbff3280132b74ded0334bed0236dae43794ad

Reply:
done in https://github.com/eclipse/mosaic/pull/138/commits/9523213c616dfcfd3fe1b1ae2d70dbe890f20ce8